example.env

```diff
@@ -1,6 +1,7 @@
 NEXMO_API_KEY='YOUR_API_KEY'
 NEXMO_API_SECRET='YOUR_API_SECRET'
 TO_NUMBER='EXAMPLE_TO_NUMBER'
+RECIPIENT_NUMBER='EXAMPLE_RECIPIENT_NUMBER'
```

Comment:
If you're going to leave in both TO_NUMBER and the preferred RECIPIENT_NUMBER, maybe stick in a comment explaining that these are the same thing, but we're moving from one to the other?